modules/xcursor.nix

```
@@ -68,7 +68,7 @@ in {
     home.packages = [ cfg.package ];
 
     xsession.initExtra = ''
-      ${pkgs.xorg.xsetroot}/bin/xsetroot -xcf ${cfg.package}/share/icons/${cfg.name}/cursors/${cfg.defaultCursor} ${
+      ${pkgs.xorg.xsetroot}/bin/xsetroot -xcf "${cfg.package}/share/icons/${cfg.name}/cursors/${cfg.defaultCursor}" ${
```

```suggestion
      ${pkgs.xorg.xsetroot}/bin/xsetroot -xcf ${cfg.package}/share/icons/${escapeShellArg cfg.name}/cursors/${escapeShellArg cfg.defaultCursor} ${
```

(Packages are top-level store paths and don't contain problematic characters)

---

I didn't know one can escape only problematic portions of the path. Thanks!